README.md

```diff
@@ -1638,6 +1638,14 @@ Specifies the name of the slot to create. Must be a valid replication slot name.
 
 This is the namevar.
 
+##### `ensure`
+
+Specifies the action to create or destroy named slot. Must be present
+
+Valid values: `present`, `absent`.
+
+Default value: `present`.
```

Comment:
The values present and absent need to have single quotes around them instead of backticks.